test/test-bot.js

```diff
@@ -0,0 +1,55 @@
+const SecretStack = require('secret-stack')
+const { createTestClient } = require('apollo-server-testing')
+
+const caps = require('ssb-caps')
+const ssbKeys = require('ssb-keys')
+const path = require('path')
+
+const GraphqlServer = require('../src/graphql-server')
+
+const DB_PATH = __dirname + '/db'
```

Comment:
is this a testbot or a bot we're gonna run for staging (like pull in all the data)
I can help clarify